iris_doc/api_tagger.py

```diff
@@ -177,7 +183,26 @@ def findFunctionParameterList(self, function_name: str, line: str) -> List[str]:
         """
         pass
 
-    def findFunctionNameFromBlock(self, block: str) -> str:
+    def matchCallbackReplacer(self, function_name: str) -> bool:
+        """
+        If this matches, we need to find the method name a different way than normal.
+        This is used for Objective-C delegate methods, for example.
+        """
+        return False
+
+    def findCallbackName(self, function_name: str, line: str) -> str:
```

Comment:
I think this function it's better defined inside the api_tagger_oc. Since the callback is one of the functions, I think it's fine to add it as a util function which helps you find it, so it's better to put it into the api_tagger_oc.